.github/workflows/test.yml

```diff
@@ -6,21 +6,61 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - uses: actions/checkout@v4
+
+      - name: Set up JDK 21
+        uses: actions/setup-java@v4
         with:
-          java-version: 1.8
+          java-version: 21
+          distribution: 'zulu'
+
       - name: Cache Maven packages
-        uses: actions/cache@v2
+        uses: actions/cache@v4
         with:
           path: ~/.m2
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
+
+      - name: Cache NVD Database
+        id: cache-nvd
+        uses: actions/cache/restore@v4
+        with:
+          path: ~/.m2/repository/org/owasp/dependency-check-data
+          key: nvd-cache-${{ runner.os }}-${{ github.run_id }}
+          restore-keys: |
+            nvd-cache-${{ runner.os }}-
```

Comment:
This has a trailing dash, is this intended?

Reply:
This is fine. It's a matching pattern